cfme/tests/automate/custom_button/test_cloud_objects.py

```diff
@@ -180,6 +180,8 @@ def test_custom_button_dialog_cloud_obj(appliance, dialog, request, setup_objs,
         1555331
         1574403
         1640592
+        1710350
```

Comment:
100 extra points if you add all of these BZ's to `automates` meta marker for the test case ... :)

Reply:
Added :). I think i need to add card as i collected mostly all BZ related to custom button from start.